portal-api/src/v1/section-currency/index.js

```diff
@@ -1,8 +1,17 @@
 const { findOneCurrency } = require('../controllers/currencies.controller');
 
+/**
+ * Find a currency by currency code and return a simple object
+ * @param currencyCode - The currency code
```

Comment:
do we need a type here {String} for currencyCode?

Reply:
I have used the same pattern as other `currencyCode` param documentations.